### PR TITLE
update infrastructure docs

### PR DESCRIPTION
**Pull request by ratik21**

### CODE REVIEW

File: infrastructure/README.md [hunk 1]
```diff
@@ -10,9 +10,8 @@ Use `make` to run tasks to `create`, `start-private-net`, `stop-private-net`.
 If you want to create a local network without using our scripts then you can follow this [tutorial](https://developer.algorand.org/tutorials/create-private-network/**.
 
 
-## Quick start
-
-**TL;DR**: Make sure you have Algorand Node (`algod` and `goal`) and `make` installed. Then `cd infrastructure; make setup-private-net`. To reset use `recreate-private-net`.
+## Quick start (local development network)
```
Comment (amityadav0) [inline infrastructure/README.md:13]: how about adding few lines about what are the benefits of using local, or sandbox ?
say, if you don't want to install algod, goal, proceed with docker

Comment (ratik21) [reply infrastructure/README.md:13]: It's already mentioned in the appropriate section - [https://github.com/scale-it/algo-builder/pull/379/files#diff-d5b352058053e1306240bcd8ab265b3234d611212cd196b2001fbb9131048566R53](https://github.com/scale-it/algo-builder/pull/379/files#diff-d5b352058053e1306240bcd8ab265b3234d611212cd196b2001fbb9131048566R53).
I prefer to keep it simple.

File: infrastructure/README.md [hunk 2]
```diff
@@ -10,9 +10,8 @@ Use `make` to run tasks to `create`, `start-private-net`, `stop-private-net`.
...
+**TL;DR**: **Make sure you have Algorand Node (`algod` and `goal`) and `make` installed**. Then `cd infrastructure; make setup-private-net`. To reset use `recreate-private-net`.
```
Comment (robert-zaremba) [inline infrastructure/README.md:14]: Please keep a new line below the header

File: infrastructure/README.md [hunk 3]
```diff
@@ -50,6 +49,14 @@ To recreate the private net (stop the current instance, remove all data and re-s
 
     make recreate-private-net
 
+## Quick start (using Sandbox)
+Algorand sandbox is a popular tool among developers to quickly setup up an algorand environment (`algod`, `indexer`, `indexer-db`) using docker. Here user don't need to explicitly setup an algorand node.
```
Comment (robert-zaremba) [inline infrastructure/README.md:53]:
```suggestion
## Quick start (using Sandbox)

Algorand sandbox is a popular tool among developers to quickly setup up an algorand environment (`algod`, `indexer`, `indexer-db`) using docker. Here user don't need to explicitly setup an algorand node.
```

**Review by robert-zaremba — APPROVED** [review]

File: infrastructure/README.md (hunk 2 above)
Comment (ratik21) [reply infrastructure/README.md:14]: yes, sorry.

**Pull request merged**